packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -106,23 +106,23 @@ export class NodeHttpRequestResponseHandler implements NodeHttpStreamsHandler {
 
     if (!nodeHttpStreams) {
 
-      logger.verbose('No node http streams received');
+      logger.debug('No node http streams received');
 
       return throwError(() => new Error('node http streams object cannot be null or undefined.'));
 
     }
 
     if (!nodeHttpStreams.requestStream) {
 
-      logger.verbose('No request stream received', nodeHttpStreams);
+      logger.debug('No request stream received', nodeHttpStreams);
 
       return throwError(() => new Error('request stream cannot be null or undefined.'));
 
     }
 
     if (!nodeHttpStreams.requestStream.headers) {
 
-      logger.verbose('No request headers received', nodeHttpStreams.requestStream);
+      logger.debug('No request headers received', nodeHttpStreams.requestStream);
```

Comment:
should these be debug, or error, since we're throwing an error after? This also seems like a big issue in the code if we don't get this.

Reply:
Fair point. However I chose debug because these cases can never occur in a real world application anyways.

Probably better to have them as `error` then by that reasonging.